src/firmware/boards/robot_stm32h7/main.c

```diff
@@ -72,18 +82,16 @@ static void MPU_Config(void);
 void MX_FREERTOS_Init(void);
 /* USER CODE BEGIN PFP */
 
-static void initIoLayer(void);
-
 /* USER CODE END PFP */
 
 /* Private user code ---------------------------------------------------------*/
 /* USER CODE BEGIN 0 */
 
-static void initIoLayer(void)
+// TODO (#2082) remove this as part of
```

Comment:
As part of what? Seems like the comment got cut off.

Reply:
fixed